debian/securedrop-export.postrm

```diff
@@ -2,6 +2,7 @@
 # postrm script for securedrop-export
 #
 # see: dh_installdeb(1)
+# shellcheck disable=SC3010
```

Comment:
Just noting for future reference, this was a mistake that's being fixed in https://github.com/freedomofpress/securedrop-client/pull/1873, shellcheck was correctly pointing out the use of bash-isms that should've either been addressed by changing the shebang to bash or switching to POSIX `test -x...` syntax.

Reply:
yeah, realized this a moment too late